stable_diffusion_text_inpaint/utils/mask_utils.py

```
@@ -0,0 +1,76 @@
+    """Create an anti-aliased mask for smoother blending.
+
+    Args:
+        image (PIL.Image): Input image
+        text_box (tuple): (x1, y1, x2, y2) coordinates for text region
+        blur_radius (int): Radius for Gaussian blur
+
+    Returns:
+        PIL.Image: Anti-aliased mask
+    """
+    mask = create_text_mask(image, text_box)
+    return mask.filter(ImageFilter.GaussianBlur(blur_radius))
```

Ruff (0.8.2)

Undefined name `ImageFilter`

(F821)